### ISSUE TITLE
Cannot flash firmware

### ISSUE THREAD

**Issue by baqwas** [edgeimpulse/firmware-sony-spresense#4]
Hello!

I cloned the repository and tried to perform the steps as documented in the section **Building and flashing the application**.

1. (`make -j`) was successful
2. Connected the board to the computer
3. Tried to flash the board; error message was:
```
arm-none-eabi-ld: Error: unable to disambiguate: -nostartfiles (did you mean --nostartfiles ?)
make: *** [Makefile:278: build/firmware.elf] Error 1
```
Is there a workaround to correct my mistake, please?

Thanks.

**Comment by mmajchrzycki** [comment]
Hi @baqwas,
Could you check if you meet all [requirements](https://github.com/edgeimpulse/firmware-sony-spresense#software)?
Also, could you write what your operating system and version are? Additional terminal output also would be helpful.

**Comment by baqwas** [comment]
Hello @mmajchrzycki,

My mistake was that I did not realize that a specific version of **Python** was needed for the install to proceed to successful conclusion. That requirement was not documented anywhere. Several helpful folks pointed out that issue to me. I logged the issue without realizing this fact since I had used other variations of the utility from the same platform for over a year now (**Ubuntu** 21.04, 21.10 & 22.04). I'm closing the issue because I cannot retro on this specific platform and **venv** is not an option for me tactically. I hope to build another crash-and-burn platform where I will install **venv** and then explore various versions of **Python** as needed. Thanks for fielding the issue.

Kind regards.
